### PR TITLE
New text on Colchicine

### PR DESCRIPTION
**Pull request by athril**
With the recent finding from COLCORONA clinical trial, I would suggest adding this drug to the manuscript 
Could you possibly review this section @rando2 @agitter ? #831

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
Huh, that's wild! Because the clinical trial with results is so small, I think this might fit better in @dianerafi's section of the discussion that discusses other possible therapeutics research could take. We could mention this and then point people to the nutraceuticals paper (@RLordan). Curious what you both think.

**Comment by athril** [comment]
There are over 4488 participants and Colchicine seems to improve the outcome, in contrast to many other drugs, e.g. hydroxychloroquine, or many other compounds that have failed in randomized clinical trials. Please let me know, what you think and if you want me to expand/modify any of the sections.

**Comment by rando2** [comment]
That preprint worries me a bit because the effect size is very small and the fact that they didn't adjust for multiple hypothesis testing (even though they note that they probably should) sets off some bells for me. I have a suspicion that paper might end up changing a bit during the peer review process! Since we're already so far over the word limit, for now I think it would be preferable to stick to adding things that have a few more trials to back them up -- obviously we didn't always do a perfect job of predicting which would pan out as the most interesting when we started back in March, but a lot of my bandwidth this week is going to making sure the sections we already have are as up to date as possible.

**Comment by RLordan** [comment]
> That preprint worries me a bit because the effect size is very small and the fact that they didn't adjust for multiple hypothesis testing (even though they note that they probably should) sets off some bells for me. I have a suspicion that paper might end up changing a bit during the peer review process! Since we're already so far over the word limit, for now I think it would be preferable to stick to adding things that have a few more trials to back them up -- obviously we didn't always do a perfect job of predicting which would pan out as the most interesting when we started back in March, but a lot of my bandwidth this week is going to making sure the sections we already have are as up to date as possible.

Colchicine is certainly gaining steam for primary/secondary treatment of CVD and there are trials underway for COVID too. I think maybe until there is another trial or two with results we should probably not make a deal of it yet. However, it is worthy of mention, maybe as @rando2 suggested we could mention it in the discussion section as a treatment coming down the line. Although not everyone is excited for colchicine https://doi.org/10.1007/s10067-020-05144-x

**Comment by dianerafi** [comment]
Hi all, please forgive my late reply to this thread, the past few days were
especially busy for me! I agree that the concept of colchicine as a
therapeutic is certainly interesting but that the effect size in this paper
is really very small - and I think too small to be considered "clinically
significant" regardless of statistical significance. A discussion of it
could fit in the section about possible future therapeutics, perhaps as a
sentence or two discussing colchicine as one example of conventional
anti-inflammatories that have potential to fight the cytokine storm in
COVID-19 (could fit in after the sentence that says "exploration of more
anti-inflammatory agents may be warranted"). That brief discussion could be
framed as a "springboard" for trying other anti-inflammatories as
therapeutics even if colchicine itself isn't the silver bullet this study
had hoped for. Let me know if I can be of any help in crafting that concept
if we ultimately decide to include it; I'm not certain how much it would
add, but I don't think it would detract from the section either.

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+P.S. Pelletier  and  J. Caventon isolated the drug in 1820 and P.L. Geiger purified its active ingredient and proposed the name colchicine in 1833 [@doi:10.2174/1871523011312010004].
+
+Colchicine is representative of organic tropones and considered a high anti-inflamatory agent [@pmc:PMC1806919], but also is a well known and powerful  spindle  poison [@doi:10.1016/s1471-4892(01)00064] with multiple side effects reported [@pmc:PMC4825447] and high toxicity in high doses [@doi:10.2174/1871523011312010004].
+Its mechanisms lie in affecing the protein threads connecting the centromere regions of chromosomes, what disrupts cell division. 
+Successful treatment of COVID-19 with colchicine can be associated with quick decrease IL‐1β, IL‐18, and IL‐6 cytokines [@doi:0.1002/jcph.1645, @doi:10.1161/JAHA.115.002128 ]. 
+
+With interest of investigating potential COVID-19 anti-inflamatory action, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033].
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:340]:
```suggestion
To investigate the potential of the anti-inflamatory properties on colchicine in COVID-19 patients, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033]. 
```

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+
+Colchicine is representative of organic tropones and considered a high anti-inflamatory agent [@pmc:PMC1806919], but also is a well known and powerful  spindle  poison [@doi:10.1016/s1471-4892(01)00064] with multiple side effects reported [@pmc:PMC4825447] and high toxicity in high doses [@doi:10.2174/1871523011312010004].
+Its mechanisms lie in affecing the protein threads connecting the centromere regions of chromosomes, what disrupts cell division. 
+Successful treatment of COVID-19 with colchicine can be associated with quick decrease IL‐1β, IL‐18, and IL‐6 cytokines [@doi:0.1002/jcph.1645, @doi:10.1161/JAHA.115.002128 ]. 
+
+With interest of investigating potential COVID-19 anti-inflamatory action, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033]. 
+In GRECCO-19, a randomized controlled trial with 105 participants (55 assigned to the colchicine group and 50 to the control group), colchicine lead to decreased  mortality  (14.0% for control group vs 1.8% in the colchicine group) and increased event-free survival (20.7 days vs 18.6) [@doi:10.1016/j.hjc.2020.03.002, doi:10.1001/jamanetworkopen.2020.13136, @clinicaltrial:NCT04326790].
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:341]:
```suggestion
In GRECCO-19, a randomized controlled trial with 105 participants (55 assigned to the colchicine group and 50 to the control group), colchicine lead to decreased mortality (14.0% for the control group vs 1.8% in the colchicine group) and increased event-free survival (20.7 days vs 18.6 days) [@doi:10.1016/j.hjc.2020.03.002, doi:10.1001/jamanetworkopen.2020.13136, @clinicaltrial:NCT04326790]. 
```

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+Its mechanisms lie in affecing the protein threads connecting the centromere regions of chromosomes, what disrupts cell division. 
+Successful treatment of COVID-19 with colchicine can be associated with quick decrease IL‐1β, IL‐18, and IL‐6 cytokines [@doi:0.1002/jcph.1645, @doi:10.1161/JAHA.115.002128 ]. 
+
+With interest of investigating potential COVID-19 anti-inflamatory action, four clinical trials have been conducted [@doi:10.1093/ehjcvp/pvaa033]. 
+In GRECCO-19, a randomized controlled trial with 105 participants (55 assigned to the colchicine group and 50 to the control group), colchicine lead to decreased  mortality  (14.0% for control group vs 1.8% in the colchicine group) and increased event-free survival (20.7 days vs 18.6) [@doi:10.1016/j.hjc.2020.03.002, doi:10.1001/jamanetworkopen.2020.13136, @clinicaltrial:NCT04326790]. 
+This evidence was further confirmed in much larget COLCORONA clinical trial [@clinicaltrial:NCT04322682].
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:342]:
```suggestion
This evidence was confirmed in the much larger COLCORONA clinical trial [@clinicaltrial:NCT04322682].
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+Among 4159 PCR confirmed COVID-19 cases, death or hospitalization due to COVID-19 infection in the 30 days following randomization appeared for 4.6% patients treated with colchicine vs 6.0% in placebo group.
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:343]:
```suggestion
Among 4,159 PCR-confirmed COVID-19 cases, death or hospitalization due to COVID-19 infection in the 30 days following randomization occurred for 4.6% patients treated with colchicine vs 6.0% in the placebo group.
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+The patients treated with colchicine experienced 25% lower hospitaliation rate, 50% less chances of necessity of mechanical ventilation and 44% lower mortality [@doi:10.1101/2021.01.26.21250494].
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:344]:
```suggestion
The patients treated with colchicine experienced 25% lower hospitaliation rate, 50% lower chance of necessity of mechanical ventilation and 44% lower mortality [@doi:10.1101/2021.01.26.21250494].
```

File: content/20.pharmaceuticals.md [hunk 6]
```diff
@@ -327,6 +327,24 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
...
+
+Two other trials are ECLA PHRI COLCOVID, which inspects co-administration of lopinavir/ritonavir [@clinicaltrial:NCT04328480], and ColCOVID-19 [@clinicaltrial:NCT04322565], have not provided their results yet.
```
Comment (cbrueffer) [inline content/20.pharmaceuticals.md:346]:
```suggestion
Two other trials are ECLA PHRI COLCOVID, which inspects co-administration of lopinavir/ritonavir [@clinicaltrial:NCT04328480], and ColCOVID-19 [@clinicaltrial:NCT04322565] are ongoing and have not reported results yet. 
```

**Review by cbrueffer — COMMENTED** [review]
Nice read; here are some suggestions for improving the text.

**Comment by athril** [comment]
A new paper in BMJ backs up studies on colchicine:
https://ard.bmj.com/content/early/2020/12/08/annrheumdis-2020-219174

**Comment by rando2** [comment]
Closing this PR due to incompatibility with the new structure of this paper

**Pull request closed without merging**